diff -up man-1.6f/src/gripes.c.loc man-1.6f/src/gripes.c
--- man-1.6f/src/gripes.c.loc	2008-06-09 14:15:14.000000000 +0200
+++ man-1.6f/src/gripes.c	2008-06-09 14:20:32.000000000 +0200
@@ -1,6 +1,7 @@
 #include <stdio.h>
 #include <stdarg.h>
 #include <stdlib.h>
+#include <locale.h>
 
 #include "gripes.h"
 #include "man.h"		/* for progname */
@@ -36,6 +37,8 @@ int cat_is_open = 0;
 static void
 catinit (void) {
     if (!cat_is_open) {
+        setenv("NLSPATH","/usr/share/locale/%l/LC_MESSAGES/%N",0);
+        setlocale(LC_ALL, "");
 #ifdef NL_CAT_LOCALE
 	catfd = my_catopen(mantexts,NL_CAT_LOCALE);
 #else
@@ -49,11 +52,11 @@ catinit (void) {
 	     */
 	    char *s, *lg;
 	    s = getenv("NLSPATH");
-	    lg = getenv("LANG");
+	    lg = getenv("LC_ALL");
 	    if (!lg)
 		    lg = getenv("LC_MESSAGES");
 	    if (!lg)
-		    lg = getenv("LC_ALL");
+		    lg = getenv("LANG");
 	    if (lg && strncmp(lg, "en", 2) && strcmp(lg, "C") && strcmp(lg, "POSIX")) {
 		fprintf(stderr,
 		  "Cannot open the message catalog \"%s\" for locale \"%s\"\n"
